frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -460,3 +470,65 @@ action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') do
 action.GitMachete.CheckoutFirstChildAction.text=Git Machete: Checkout First Child Branch
 action.GitMachete.CheckoutFirstChildAction.description=Checkout first child branch (''{0}'')
 action.GitMachete.CheckoutFirstChildAction.undefined.branch-name=Branch (''{0}'') does not have any children in the branch layout
+
+# Traverse
+
+action.GitMachete.BaseTraverseAction.description=Traverse
+action.GitMachete.BaseTraverseAction.description.empty-layout=Cannot traverse an empty branch layout
+
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title=Traverse Info
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML=<html>\
+  This action triggers an interactive walk over the branches and suggests actions (pushes/pulls/rebases) \
+  necessary to achieve both child-to-parent and local-to-remote synchronization.<br><br>\
+  Do you want to proceed?\
+  </html>
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.text.HTML=\
+  <html>\
+    <p>The branch <b>{0}</b> is diverged from remote <b>{1}</b>, and it is <b>{2}</b> remote.</p> \
+    <p>Please choose the preferred action for syncing it:</p>\
```

Comment:
Okay... so the change with opinionated action on diverged-from-remote is still pending/TODO right?

Reply:
nope, I simply missed this one - the message should be different in the case of diverged and newer ofc